code/go/0chain.net/smartcontract/dbs/event/edb_transaction.go

```diff
@@ -0,0 +1,15 @@
+package event
+
+import (
+	"0chain.net/smartcontract/dbs"
+	"gorm.io/gorm"
+)
+
+type edbTransaction struct {
+	dbs.Store
+	tx *gorm.DB
+}
+
+func (tx edbTransaction) Get() *gorm.DB {
```

Comment:
also switch off default transaction mode
```
tx := db.Session(&Session{SkipDefaultTransaction: true})

```

Reply:
We can make using transactions optional by adding a `use_transactions` option to `0chian.yaml`. Why do we want this option?